### PR TITLE
refactor(protocol-designer,labware-creator): remove favicon generator dep in favor of static favicon

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

`favicons-webpack-plugin` depends on `sharp` which is not compatible with node version 14.  We need to upgrade to node version 14 for #8226 . Remove this bundle time dependency for now. We should look to replace it down the road, but favicon generation should not be a road block to bigger changes.

# Changelog

- remove `favicons-webpack-plugin` package and usage from LL and PD webpack configs
- replace with static reference to favicon logo image using `html-webpack-plugin`'s `favicon` option

# Review requests

- does this look acceptable?

# Risk assessment
low, favicons in general are a pretty low concern, the only use cases where we are degrading appearance slightly here are browsers that may render favicons as full thumbnails. These will appear pixelated for PD and LL.  This is pretty common for other websites.

### CODE REVIEW

File: labware-library/webpack.config.js [hunk 1]
```diff
@@ -54,23 +53,7 @@ module.exports = merge(baseConfig, {
...
-        android: false,
-        appleIcon: false,
-        appleStartup: false,
-        coast: false,
-        favicons: true,
-        firefox: false,
-        windows: false,
-        yandex: false,
-      },
+      favicon: './src/images/favicon-logo.png',
```
Comment (mcous) [inline labware-library/webpack.config.js]: I wonder if it's a problem that this file is not in a "standard" favicon size? Seems to be working in the dev server

**Review by mcous — COMMENTED** [review]
Looks like there's an old (manual) reference to a favicon in `labware-library/src/index.hbs` that needs to be removed, too

File: labware-library/webpack.config.js (hunk 1 above)
Comment (b-cooper) [reply labware-library/webpack.config.js]: I've replaced it with a favicon.ico that wraps up 16x16, 32x32, and 64x64 (which should be a much more standard set of supported sizes)

**Review by mcous — APPROVED** [review]

**Review by sfoster1 — APPROVED** [review]
Based on CI for #8317 this will work great!

**Pull request merged**